Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -5,24 +5,214 @@
 import Foundation
 import UIKit
 
-private let StatusBarHeight = 20
+private let StatusBarHeight = CGFloat(20)
+private let TextBoxHeight = CGFloat(25.0)
+private let Margin = CGFloat(10)
+private let ToolbarHeight: CGFloat = 44
+private let CellHeight = TextBoxHeight * 4
+
+@objc
+class TransitionOptions {
```

Comment:
Couple of thoughts on this class:
* Do we need it at all? We already get a nice transitionContext from the UIViewControllerAnimatedTransitioning protocol. Can't we just forward this context along?
* Having a String-keyed dict for this seems like overkill, and we get less compiler checks making mistakes easier. If you don't want to get rid of this class, can you just explicitly add the option fields as part of this class (i.e., the same way the original transitionContext is defined)? Currently, it looks like that would just be a single <code>container: UIView?</code>.